doc/changelog.md

```diff
@@ -1,6 +1,7 @@
 # ChangeLog
 
 * **0.33.0** (2020-01-21)
+  - Handle authentication tokens returned from credential helpers ([#1348](https://github.com/fabric8io/docker-maven-plugin/issues/1348))
```

Comment:
Please move that up to a **0.33-SNAPSHOT** section as this will become the changes for the next release. 0.33.0 is already released.

Reply:
Done!